docs/anomaly-detection/QuickStart.md

```diff
@@ -0,0 +1,103 @@
+---
+sort: 1
+weight: 1
+title: VictoriaMetrics Anomaly Detection Quick Start
+menu:
+  docs:
+    parent: "anomaly-detection"
+    weight: 1
+    title: Quick Start
+aliases:
+- /anomaly-detection/QuickStart.html
+---
+
+# VictoriaMetrics Anomaly Detection Quick Start
+
+It is recommended to read [README](https://docs.victoriametrics.com/anomaly-detection/)
+and [Overview](https://docs.victoriametrics.com/anomaly-detection/overview.html)
+before you start working with `vmanomaly`.
+
+## How to install and run `vmanomaly`
+
+>`vmanomaly` is a part of VictoriaMetrics Enterprise version. You can get a license key [**here**](https://victoriametrics.com/products/enterprise/trial/index.html).
```

Comment:
explain that for running it you need to have license key, trial one can be issued via link ....